packages/fxa-settings/src/lib/gql.ts

```diff
@@ -46,7 +38,7 @@ export const errorHandler: ErrorHandler = ({ graphQLErrors, networkError }) => {
       reauth = true;
     }
   }
-  if (reauth && !currentPageDoesNotRequireAuthentication) {
+  if (reauth && currentPageRequiresAuthentication) {
     window.location.replace(
       `/signin?redirect_to=${encodeURIComponent(window.location.pathname)}`
     );
```

Comment:
Should the `else` statement only apply to `currentPageRequiresAuthentication`?

Reply:
Hmm...I'm not sure actually. What do you think?

Reply:
I'm not sure, that's why I asked 😁 Since we currently expect an error for unauthenticated pages, we might not want to console log the error in that situation?

Reply:
Oh I see what you mean, I'll add that exception